virttest/guest_agent.py

```diff
@@ -509,7 +509,11 @@ def shutdown(self, mode=SHUTDOWN_MODE_POWERDOWN):
         if mode in [self.SHUTDOWN_MODE_POWERDOWN, self.SHUTDOWN_MODE_REBOOT,
                     self.SHUTDOWN_MODE_HALT]:
             args = {"mode": mode}
-        self.cmd(cmd=cmd, args=args, success_resp=False)
+        try:
+            self.cmd(cmd=cmd, args=args, success_resp=True)
+        except VAgentCmdError as e:
+            str(e)
+            return False
```

Comment:
Based on https://github.com/autotest/tp-qemu/pull/1032
```python
        except VAgentCmdError:
            return False
```
would be enough.

Can you explain what `str(e)` is for on line 515

Reply:
@hereischen You are right,  Don't need "str(e)". (original idea was to use str(e) to call   __str__(self) method in Class VAgentCmdError).